core/modules/user/user.admin.inc

```diff
@@ -661,13 +661,23 @@ function user_admin_role($form, &$form_state, $role = NULL) {
   $form['actions']['submit'] = array(
     '#type' => 'submit',
     '#value' => t('Save role'),
+    '#weight' => 2,
   );
+  // Only show the 'Save and set permissions' button when creating a new role.
+  if (empty($role)) {
+    $form['actions']['continue'] = array(
+      '#type' => 'submit',
+      '#value' => t('Save and set permissions'),
+      '#weight' => 1,
```

Comment:
If we move this button above the `Save role` button we won't need the weights.

Reply:
Yes 👍🏼 ...fixed with [`11b1eee` (#3728)](https://github.com/backdrop/backdrop/pull/3728/commits/11b1eee42dcac2f37c58368a596ad72f72cee4be)